### PR TITLE
Accessibility Improvements and general refactor

### LINKED ISSUE #746: Misc updates to improve admin experience
- [x] ~Turn off block editor for sponsors post type as editor is not used~ - won't do
- [x] Add columns to speakers post type to show custom field status

### LINKED ISSUE #749: Turn off XML-RPC
Just creating this issue to track findings around XML-RPC

From WP Scan:

[+] XML-RPC seems to be enabled: https://events.linuxfoundation.org/wp/xmlrpc.php
 | Found By: Direct Access (Aggressive Detection)
 | Confidence: 100%
 | References:
 |  - http://codex.wordpress.org/XML-RPC_Pingback_API
 |  - https://www.rapid7.com/db/modules/auxiliary/scanner/http/wordpress_ghost_scanner/
 |  - https://www.rapid7.com/db/modules/auxiliary/dos/http/wordpress_xmlrpc_dos/
 |  - https://www.rapid7.com/db/modules/auxiliary/scanner/http/wordpress_xmlrpc_login/
 |  - https://www.rapid7.com/db/modules/auxiliary/scanner/http/wordpress_pingback_access/

### LINKED ISSUE #751: Remove WordPress version from styles and scripts versions
WordPress version 6.1.1 identified (Latest, released on 2022-11-15).
 | Found By: Most Common Wp Includes Query Parameter In Homepage (Passive Detection)
 |  - https://events.linuxfoundation.org/wp/wp-includes/css/dist/block-library/style.min.css?ver=6.1.1

+] WordPress version 6.1.1 identified (Latest, released on 2022-11-15).
 | Found By: Query Parameter In Install Page (Aggressive Detection)
 |  - https://pr-747-lfeventsci.pantheonsite.io/wp/wp-includes/css/dashicons.min.css?ver=6.1.1
 |  - https://pr-747-lfeventsci.pantheonsite.io/wp/wp-includes/css/buttons.min.css?ver=6.1.1
 |  - https://pr-747-lfeventsci.pantheonsite.io/wp/wp-admin/css/forms.min.css?ver=6.1.1
 |  - https://pr-747-lfeventsci.pantheonsite.io/wp/wp-admin/css/l10n.min.css?ver=6.1.1
 |  - https://pr-747-lfeventsci.pantheonsite.io/wp/wp-admin/css/install.min.css?ver=6.1.1

### PR DESCRIPTION
**Pull request by cjyabraham**
- Accessibility improvements, see issue #673
- Does a big refactor of theme code into the MU plugin
- Splits the MU plugin into different files, ala CNCF
- Disables xmlrpc in order to reduce security attack surface and performance degradation
- Closes #751 
- Closes #749 
- Closes #746 

[Dev instance](https://pr-747-lfeventsci.pantheonsite.io/)

On deploy:
- [ ] Configure PublishPress to insist on image alt tags prior to publishing.
- [ ] Update all HubSpot newsletter forms to have labels
- [ ] Re-run page speed tests
- [ ] Re-save image blocks where we want them to be fully accessible

### CODE REVIEW

**Comment by cjyabraham** [comment]
For the work so far:

**Before**
Screen Shot 2023-03-11 at 11 58 09 AM

**After**
Screen Shot 2023-03-11 at 11 58 14 AM

**Comment by thetwopct** [comment]
#Before 

Using pagespeed.web.dev on the current site - **home page**:

Screenshot-2023-03-15 --00 56 12@2x

Screenshot-2023-03-15 --00 56 23@2x

Using pagespeed.web.dev on the current site - **KCCNC EU 23**:

Screenshot-2023-03-15 --00 54 48@2x

Screenshot-2023-03-15 --00 55 30@2x

File: web/wp-content/themes/lfevents/src/assets/scss/components/_forms.scss [hunk 1]
```diff
@@ -105,3 +105,13 @@ div.hs_recaptcha,
   width: 1px;
   overflow: hidden;
 }
+
+.searchandfilter .sf-input-text {
+  &:focus,
+  &:focus-within {
```
Comment (cjyabraham) [inline web/wp-content/themes/lfevents/src/assets/scss/components/_forms.scss:111]: Ooh `focus-within`! That's a new one for me!

**Comment by cjyabraham** [comment]
I was looking into the request for Kubecon NA to show different images in the hero of the site depending on whether it’s desktop or mobile. My experience is that it currently works on the dev site but does not work on this PR, with the newly updated conditional blocks pro plugin. I have reported the bug to the plugin author as follows:

```
We just upgraded to version 3.0.1 and the screen size conditions no longer seem to do anything on the site. They used to work fine with the previous version we were running, version 2.6.0. Any ideas on how to fix?
```

**Comment by cjyabraham** [comment]
I have rolled back to the old version of conditional blocks pro as the new version has a bug

**Comment by cjyabraham** [comment]
# After 

Using pagespeed.web.dev on the current site - **home page**:
Screen Shot 2023-03-27 at 12 57 08 PM
Screen Shot 2023-03-27 at 12 57 15 PM

Using pagespeed.web.dev on the current site - **KCCNC EU 23**:

Screen Shot 2023-03-27 at 1 00 41 PM
Screen Shot 2023-03-27 at 1 00 47 PM

**Pull request merged**